sites/fast-creator/specs/site.spec.md

```diff
@@ -0,0 +1,22 @@
+# Creator specifications
+
+This file contains all specifications in regards to the Creator site's architecture and planning. Additional information may be referred to from supplemental markdown files.
+
+- [Projects](#projects)
+    - [Web component libraries](#web-component-libraries)
+
+## Projects
+
+Projects are the serializable data that users create and modify in the FAST Creator site. They consist of multi-page layouts representing sites.
```

Comment:
Potential tangent 😄 - Is FAST Creator better described as a web app?

Reply:
Web app is probably more precise even though I believe web apps fall into the same bucket as web sites. I'll change that so it's more apparent that it has more interactivity one would assume for a web app.